source/tools/testConsistency/dotnetWPF-renderer/WPFRenderer/WPFRenderer/Program.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using AdaptiveCards;
+using AdaptiveCards.Rendering.Wpf;
+using AdaptiveCards.Templating;
+using Newtonsoft.Json.Linq;
+
+
+namespace WPFRenderer
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            JObject resultJson = new JObject();
+            List<string> errorList = new List<string>();
+            List<string> warningList = new List<string>();
+            resultJson.Add("imageData", "");
+            resultJson.Add("warnings", "");
+            resultJson.Add("errors", "");
+            Console.OutputEncoding = Encoding.UTF8;
+            try
+            {  
+                string inputString = "";
+                string currLine;
+            
+                //read a multi line input from stdin
+                do
+                {
+                    currLine = Console.ReadLine();
+                    inputString = string.Concat(inputString, currLine);
+                }
+                while ((currLine != null) && (currLine != ""));
+
+                JObject jsonInput = JObject.Parse(inputString);
+                string cardData = jsonInput.GetValue("data").ToString();
+                string cardTemplate = jsonInput.GetValue("template").ToString();
+          
+                AdaptiveCardTemplate adaptiveCardTemplate = new AdaptiveCardTemplate(cardTemplate);
+                string adaptiveCardJson = adaptiveCardTemplate.Expand(cardData);
+                AdaptiveCardParseResult parseResult = AdaptiveCard.FromJson(adaptiveCardJson);
+                AdaptiveCardRenderer renderer = new AdaptiveCardRenderer();
+
+                GetResultJsonString(parseResult, renderer, resultJson, warningList).GetAwaiter().GetResult();
+           
+            }
+
```

Comment:
nit: there shouldn't be additional newlines between each catch statement

Reply:
@paulcam206 what's our story/guidance for .NET so far? I was trying to look at our existing .editorconfigs and looks like .NET doesnt have any defined yet..Do we have stylecop running in teh meanwhile (stylecop.analyzers nuget package).. Or adding an .editorconfig for .NET following [these docs](https://docs.microsoft.com/en-us/visualstudio/ide/code-styles-and-code-cleanup?view=vs-2019)